content/docs/iac/support/version-support.md

```diff
@@ -0,0 +1,43 @@
+---
+title_tag: "Pulumi Provider Version Support"
+meta_desc: "Learn best practices for maintaining up-to-date pulumi applications"
+title: Pulumi Provider Version Support
+h1: Pulumi Provider Version Support
+meta_image: /images/docs/meta-images/docs-meta.png
+weight: 6
+menu:
+    iac:
+        name: Version Support
+        parent: iac-support
+        weight: 6
+---
+
+
+Pulumi focuses on actively supporting the latest released version of the providers we maintain.
+While we do not formally designate older versions for long term support,
+we are committed to helping our users succeed and continuously improve our providers based on community feedback and contributions.
+We encourage users to update to the latest versions to benefit from the most recent features, bug fixes, and security enhancements.
+You can find the current version for all providers maintained by Pulumi in the [Pulumi Registry](https://www.pulumi.com/registry/)
```

Comment:
@mjeffryes Did you intend to include a line break with each new sentence? Markdown renders it as a single paragraph, but typically, for maintainability reasons, I think it's better to include the entire paragraph on one line and let your preferred IDE handle the word wrap.